Source/CompanyCommunicator.Prep.Func/Export/Streams/DataStreamFacade.cs

```diff
@@ -121,8 +139,8 @@ await foreach (var sentNotificationDataEntities in sentNotificationDataEntitiesS
                     {
                         Id = sentNotificationDataEntity.RowKey,
                         Name = team?.Name,
-                        DeliveryStatus = this.localizer.GetString(sentNotificationDataEntity.DeliveryStatus),
-                        StatusReason = this.GetStatusReason(sentNotificationDataEntity.ErrorMessage, sentNotificationDataEntity.StatusCode.ToString()),
+                        DeliveryStatus = sentNotificationDataEntity.DeliveryStatus is null ? sentNotificationDataEntity.DeliveryStatus : this.localizer.GetString(sentNotificationDataEntity.DeliveryStatus),
```

Comment:
Can this be `x.DeliveryStatus ?? y`?

Reply:
As discussed, the scenario here is if null then null else localizer.